src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -478,11 +478,21 @@ private function getOrderDocuments(Order $order): OrderDocumentsForViewing
                     $conf[$this->contextLanguageId] ?? '',
                     $document->id
                 );
+
+                $total_cart_rule = 0;
+                $cart_rules = $order->getCartRules();
+
+                if ($document->order_slip_type == 1 && is_array($cart_rules)) {
+                    foreach ($cart_rules as $cart_rule) {
+                        $total_cart_rule += $cart_rule['value'];
```

Comment:
Are you sure 'value' key is set in `$cart_rule` here ?

Reply:
Yep it is:

![image](https://user-images.githubusercontent.com/48719560/177109950-3296de00-55a6-46c6-bf33-f4b5a047ff1e.png)